contracts/cryptoblades.sol

```diff
@@ -271,6 +271,49 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
         require(foundMatch, "Target invalid");
     }
 
+    mapping (address => uint32) tickets;
```

Comment:
This should be added in a brand-new contract instead of the main game contract. I'll have to get back to you whether I just make those myself, let another PR create them or have this PR create them.